addons/hr_recruitment/wizard/applicant_send_mail_views.xml

```diff
@@ -9,20 +9,20 @@
                 <field name="render_model" invisible="1"/>
                 <field name="template_id" invisible="1"/>
                 <group>
+                    <field string="To" name="applicant_ids" widget="many2many_tags" context="{'show_partner_name': 1}"/>
                     <field name="subject" required="1"/>
-                    <field name="applicant_ids" widget="many2many_tags" context="{'show_partner_name': 1}"/>
                 </group>
                 <field name="body" nolabel="1" class="oe-bordered-editor"
                         widget="html_mail"
                         placeholder="Write your message here..."
                         force_save="1"/>
-                <group>
-                            <field name="attachment_ids" widget="many2many_binary" string="Attach a file" nolabel="1" colspan="2"/>
-                            <field name="template_id" string="Load template" options="{'no_create': True}"/>
-                </group>
+                <field name="attachment_ids" widget="mail_composer_attachment_list"/>
                 <footer>
                     <button name="action_send" string="Send" type="object" class="btn-primary" data-hotkey="q"/>
                     <button string="Cancel" class="btn-secondary" special="cancel" data-hotkey="x"/>
+                    <field name="attachment_ids" widget="mail_composer_attachment_selector"/>
+                    <field name="template_id" widget="mail_composer_template_selector" options="{'no_create': True}"/>
```

Comment:
It seems that the `mail_composer_template_selector` widget doesn't have a `no_create` option. Is it still necessary?
I don't know what should be the behaviour though. If we want to prevent using the "save as template" feature, then the widget should be modified. But my feeling is that the "save as template" feature is quite different from the many2one quick creation feature and could be used here.